Python Agent: Add callout for High Security Mode in affected settings (#18025)
newrelic/docs-website: src/content/docs/apm/agents/python-agent/configuration/python-agent-configuration.mdx

    </table>

    When set to `true`, enables AI monitoring.

    <Callout variant="important">

>       This setting is disabled when [High Security Mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.

Reviewer: ```suggestion
      This setting is disabled when [high security mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.
```

    When the transaction tracer is [enabled](#txn-tracer-enabled), the agent can record SQL statements. The recorder has three modes: `off` (sends no SQL), `raw` (sends the SQL statement in its original form), and `obfuscated` (strips out numeric and string literals).

    Most web frameworks (including Django) parameterize SQL queries so they do not actually contain the values used to fill out the query. If you use `raw` mode with one of these frameworks, the Python agent will only see the SQL prior to insertion of values. The parametrized SQL will look much like `obfuscated` mode.
  
    <Callout variant="important">

>       This setting is set to `obfuscated` when [High Security Mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.

Reviewer: ```suggestion
      This setting is disabled when [high security mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.
```

    </table>

    Allow recording of events to the Event API via [`record_custom_event()`](/docs/agents/python-agent/python-agent-api/recordcustomevent-python-agent-api/).

    <Callout variant="important">

>       This setting is disabled when [High Security Mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.

Reviewer: ```suggestion
      This setting is disabled when [high security mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.
```

    If `true`, the agent captures log records emitted by your application and forwards them to New Relic. `application_logging.enabled` must also be `true` for this setting to take effect.

    <Callout variant="important">

>       This setting is disabled when [High Security Mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.

Reviewer: ```suggestion
      This setting is disabled when [high security mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.
```

    </table>

    Set to `true` to enable capturing of the raw inference value.

    <Callout variant="important">

>       This setting is disabled when [High Security Mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.

Reviewer: ```suggestion
      This setting is disabled when [high security mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.
```

    </table>

    Allow recording of machine learning events to the Event API via [`record_ml_event()`](/docs/agents/python-agent/python-agent-api/recordmlevent-python-agent-api/).
  
    <Callout variant="important">

>       This setting is disabled when [High Security Mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.

Reviewer: ```suggestion
      This setting is disabled when [high security mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.
```

    </table>

    If enabled, exception messages will be stripped from error traces before they are sent to the [collector](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#collector), in order to prevent the inadvertent capture of sensitive information. This option is automatically enabled in high-security mode.

    <Callout variant="important">

>       This setting is enabled when [High Security Mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.

Reviewer: ```suggestion
      This setting is disabled when [high security mode](/docs/apm/agents/python-agent/getting-started/apm-agent-security-python/#restricted) is enabled.
```
